deployment/embedded-spring-rest/README.md

```diff
@@ -1,9 +1,9 @@
-# Spring configured embedded process engine and REST API
+# Spring 5 configured embedded process engine and REST API
```

Comment:
🔧 Add `5` to the `Spring` keyword where applicable in https://github.com/camunda/camunda-bpm-examples#deployment--project-setup-examples